build.xml

```diff
@@ -28,15 +28,12 @@
   <property name="compile.java.bootclasspath" value="${build.bootclasspath}"/>
 
   <path id="adaptor.build.classpath">
-<!--
```

Comment:
That's my best archaeological guess. No, the only JAR file in lib is JUnit. Plus, note that this is commented out. Yes, a FileSet with no includes or excludes selects everything in the directory.